Google.PowerShell/Compute/GceForwardingRuleCmdlets.cs

```diff
@@ -0,0 +1,174 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
+// Licensed under the Apache License Version 2.0.
+
+using Google.Apis.Compute.v1;
+using Google.Apis.Compute.v1.Data;
+using Google.PowerShell.Common;
+using System.Collections.Generic;
+using System.Management.Automation;
+
+namespace Google.PowerShell.ComputeEngine
+{
+    /// <para type="synopsis">
+    /// Gets Google Compute Engine forwarding rules.
+    /// </para>
+    /// <para type="description">
+    /// Lists forwarding rules of a project, or gets a specific one.
+    /// </para>
+    /// <example>
+    /// <code>PS C:\> Get-GceForwardingRule</code>
+    /// <para>Lists all forwarding rules for the default project.</para>
+    /// </example>
+    /// <example>
+    /// <code>PS C:\> Get-GceForwardingRule -Region us-central1</code>
+    /// <para>Lists all forwarding rules in region "us-central1" for the default project.</para>
+    /// </example>
+    /// <example>
+    /// <code>PS C:\> Get-GceForwardingRule "my-forwarding-rule"</code>
+    /// <para>Gets the forwarding rule named "my-forwarding-rule" in the default project and region</para>
+    /// </example>
+    /// <example>
+    /// <code>PS C:\> Get-GceForwardingRule -Project my-project -Global</code>
+    /// <para>Lists all global forwarding rules for the project named "my-project".</para>
+    /// </example>
+    /// <example>
+    /// <code>PS C:\> Get-GceForwardingRule "my-forwarding-rule" -Gobal</code>
+    /// <para>Gets the global forwarding rule named "my-forwarding-rule" in the default project</para>
```

Comment:
End this snippet with a period, and all previous text snippets as well. Is there any reason not to use complete senences?